src/assets/styles/styles/custom/_tables.scss

```diff
@@ -32,35 +32,37 @@
       border: 1px solid $table-border-color-grey;
       align-items: center;
       width: 120%;
-      height: 103%;
+      height: 101%;
```

Comment:
wizualnie wydawało się, że weekend się rozjeżdaz (tak na prawdę nie, ale inny kolor tła to robił)
przeszkadzało mi to i zmieniłąm

ta zmiana powoduje, ze top i botton tabelki jest szerszy niż powinien